test/functional/verify/picky-ghosted-missing/test.bats

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bats
+
+load "../../swupdlib"
+
+setup() {
+  clean_test_dir
+  create_manifest_tar 10 MoM
+  sign_manifest_mom 10
+  create_manifest_tar 10 os-core
+  chown_root -R "$DIR/target-dir"
+}
+
+teardown() {
+  clean_tars 10
+  sudo cp -r "$DIR/target-dir.bak/usr" "$DIR/target-dir/"
+}
+
+@test "verify ghosted file not added during --picky" {
+  run sudo sh -c "$SWUPD verify --fix --picky $SWUPD_OPTS"
+
+  check_lines "$output"
+  [ $status -eq 0 ]
+  # this should exist at the end, despite being marked as "ghosted" in the
+  # Manifest. This is an important test because internally swupd treats ghosted
+  # files as deleted for rename purposes, but we need to make sure the file
+  # doesn't actually get deleted.
```

Comment:
This comment needs updating to describe the fact the verify will not install a missing file if it is ghosted -- and no renames are involved for this test.